jkube-kit/enricher/api/src/test/java/org/eclipse/jkube/kit/enricher/handler/PodTemplateHandlerTest.java

```diff
@@ -168,9 +163,9 @@ public void podWithInvalidVolumeTypeTemplateHandlerTest(){
         PodTemplateSpec podTemplateSpec = podTemplateHandler.getPodTemplate(config, null, images);
 
         //Assertion
-        assertEquals("test-account",podTemplateSpec.getSpec().getServiceAccountName());
-        assertTrue(podTemplateSpec.getSpec().getVolumes().isEmpty());
-        assertNotNull(podTemplateSpec.getSpec().getContainers());
+        assertThat(podTemplateSpec.getSpec().getServiceAccountName()).isEqualTo("test-account");
+        assertThat(podTemplateSpec.getSpec().getVolumes()).isEmpty();
+        assertThat(podTemplateSpec.getSpec().getContainers()).isNotNull();
```

Comment:
Please chain assertions here. Try to do assertion using one `assertThat` statement like done in other places.